pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -92,6 +93,15 @@ func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, includeInputImag
 		return "", NotFoundErr(fmt.Sprintf("%q", ist.Name))
 	}
 
+	markers := buildanalysis.FindCircularBuilds(g, osgraph.DefaultNamer)
+	if len(markers) > 0 {
+		for _, marker := range markers {
+			if strings.Contains(marker.Message, ist.Name) {
+				return fmt.Sprintf("Exiting examination of %s. %s", ist.Name, marker.Message), nil
```

Comment:
no i'm just asking you to show me the full output from oc status so i can see where your new message fits in w/ the existing output.